blocks/api/registration.js

```diff
@@ -38,6 +38,19 @@ let unknownTypeHandlerName;
  */
 let defaultBlockName;
 
+/**
+ * Constant mapping post formats to the expected default block.
+ *
+ * @type {Object}
+ */
+const POST_FORMAT_BLOCK_MAPPER = {
```

Comment:
Minor: "mapper" implies that it performs some action, when in fact it's the map(ping) itself.